{"name": "test_deletion_without_id", "status": "failed", "statusDetails": {"message": "AssertionError: Status code 400 was expected and code 404 was returned.\nassert 404 == 400\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test.test_CourierDelete.TestDeleteCourier object at 0x00000228F2D08390>, base_url = 'http://qa-scooter.praktikum-services.ru/api/v1/'\n\n    def test_deletion_without_id(self, base_url):\n        logger.info(\"Testing courier deletion without ID\")\n        response = requests.delete(base_url + f\"courier/\")\n        logger.debug(f\"Status code {response.status_code} was returned and message{response.json()}\")\n>       assert response.status_code == 400, f\"Status code 400 was expected and code {response.status_code} was returned.\"\nE       AssertionError: Status code 400 was expected and code 404 was returned.\nE       assert 404 == 400\nE        +  where 404 = <Response [404]>.status_code\n\ntest\\test_CourierDelete.py:26: AssertionError"}, "attachments": [{"name": "log", "source": "8bf46166-20a5-47dc-b75d-fc7260362947-attachment.txt", "type": "text/plain"}], "start": 1692635388414, "stop": 1692635388664, "uuid": "94bf76e5-fe3c-4bf2-abaa-1252da737b1c", "historyId": "25d7ffea571398379f3339ab13d304ac", "testCaseId": "25d7ffea571398379f3339ab13d304ac", "fullName": "test.test_CourierDelete.TestDeleteCourier#test_deletion_without_id", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_CourierDelete"}, {"name": "subSuite", "value": "TestDeleteCourier"}, {"name": "host", "value": "DESKTOP-36NVPK8"}, {"name": "thread", "value": "8420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_CourierDelete"}]}